### ISSUE TITLE
help for retrainning model 

### ISSUE THREAD

**Issue by nanoimoretti** [Shaon2221/Livestock-Cattle-Weight-Estimation#1]
Hello my name is Ignacio, I'm trying the app and it seems very good. I have an image bank with which I would like to retrain the model, I wanted to ask you if there is any guide on how to retrain it? thank you very much.

**Comment by RitikGandhi21** [comment]
> Hello my name is Ignacio, I'm trying the app and it seems very good. I have an image bank with which I would like to retrain the model, I wanted to ask you if there is any guide on how to retrain it? thank you very much.

Hi Ignacio
Are you specifically working on cattle weight estimation or do you want to retrain it for a different dataset?

**Comment by nanoimoretti** [comment]
Thank you very much for the prompt reply, I am indeed working on weight prediction of Argentinean cattle. The main breeds are Angus, Hereford, Brahman, Brangus, Braford, Limousin and Criollo. I have a dataset of images annotated with actual weights and I would like to train this model with that dataset.

**Comment by RitikGandhi21** [comment]
I am also planning to retrain the model. Have you tried training it on any other models? If yes could you please share a link to those models?

Thank you!

**Comment by nanoimoretti** [comment]
Yes, I am currently trying to reproduce these 3 codes to test different models. 

https://www.kaggle.com/datasets/sadhliroomyprime/cattle-weight-detection-model-dataset-12k

https://github.com/nayemabs/cattle-web

https://github.com/Parit99/Computer-Vision-Weight-Prediction

**Comment by RitikGandhi21** [comment]
I appreciate your prompt reply, Ignacio. If it's convenient for you, may we shift our conversation to email? I've been exploring some ideas inspired by the papers I've been reading and would be delighted to share and discuss them with you. Additionally, I have plans to experiment with a few models. If this piques your interest, please feel free to reach out to me at ritikgandhi21@gmail.com

**Comment by thalisvilela** [comment]
Can i join you guys in this conversation? I'm really interested in this topic, as i'm working with some cattle industries here in Brazil on evaluating the feasibility and readiness of this type of solution.

**Comment by RitikGandhi21** [comment]
> Can i join you guys in this conversation? I'm really interested in this topic, as i'm working with some cattle industries here in Brazil on evaluating the feasibility and readiness of this type of solution.

Sure, have you tried any approach yet?
